spec/tasks/gen/action_spec.cr

```diff
@@ -35,16 +47,28 @@ describe Gen::Action do
   end
 
   it "displays an error if given only one class" do
-    begin
+    with_cleanup do
       io = IO::Memory.new
-      ARGV.push("Users")
+      invalid_action_name = "Users"
+      ARGV.push(invalid_action_name)
 
       Gen::Action.new.call(io)
-      message = "\e[31mThat's not a valid Action.  Example: lucky gen.action Users::Index\e[0m"
+      message = "\e[31mThat's not a valid Action. Example: lucky gen.action Users::Index\e[0m"
 
       io.to_s.strip.should eq(message)
```

Comment:
You could use `contain` here as well so you don't need the color stuff at the beginning and end. Maybe something like: `io.to_s.should contain "That's not a valid Action"`